upup/pkg/fi/cloudup/awstasks/dnsname.go

```diff
@@ -196,7 +198,7 @@ func (e *DNSName) Run(c *fi.Context) error {
 
 func (s *DNSName) CheckChanges(a, e, changes *DNSName) error {
 	if a == nil {
-		if fi.StringValue(e.Name) == "" {
+		if fi.StringValue(e.ResourceName) == "" {
 			return fi.RequiredField("Name")
```

Comment:
```suggestion
			return fi.RequiredField("ResourceName")
```

Do we not also want to make `Name` required?